designsafe/static/scripts/projects/components/edit-project/edit-project.component.js

```diff
@@ -352,17 +352,9 @@ class EditProjectCtrl {
             projectData.keywords = this.form.keywords;
         }
         this.savePrj(projectData).then((project) => {
+            this.project.value = project.value;
             this.close({$value: project});
             this.ui.busy = false;
-            this.$state.go(
```

Comment:
Is this going to affect project creation?

Reply:
Ah yeah, I was not thinking of new projects. I thought this was the redirect after changing the project type. I'll get this sorted out